tests/test_base.py

```diff
@@ -1,6 +1,7 @@
 import os
 import telnetlib
 import unittest
+from unittest.mock import patch
```

Comment:
can you please do:

```
from unittest import mock
```

And avoid changing the other lines? I think it's easier to read.

Reply:
Review Please :smile: